doc/parameters/param_list.md

```diff
@@ -102,328 +101,332 @@ Namespace: /parent_ns/local_ns
+| `<dwb plugin>`.decel_lim_x | 0.0 | Maximum deceleration X (m/s^2) |
+| `<dwb plugin>`.decel_lim_y | 0.0 | Maximum deceleration Y (m/s^2) |
+| `<dwb plugin>`.decel_lim_theta | 0.0 | Maximum deceleration rotation (rad/s^2) |
 
-### xy_theta_iterator 
+## xy_theta_iterator 
 
 | Parameter | Default | Description |
 | ----------| --------| ------------|
-| `<dwb plugin>`.vx_samples | 20 | |
-| `<dwb plugin>`.vy_samples | 5 | |
-| `<dwb plugin>`.vtheta_samples | 20| |
+| `<dwb plugin>`.vx_samples | 20 | Number of  velocity samples in the X velocity direction |
+| `<dwb plugin>`.vy_samples | 5 | Number of velocity samples in the Y velocity direction |
+| `<dwb plugin>`.vtheta_samples | 20 | Number of velocity samples in the angular directions |
 
-### base_obstacle TrajectoryCritic
+## base_obstacle TrajectoryCritic
 
 * `<name>`: base_obstacle critic name defined in `<dwb plugin>.critics`
 
 | Parameter | Default | Description |
 | ----------| --------| ------------|
-| `<dwb plugin>`.`<name>`.sum_scores | false | |
-| `<dwb plugin>`.`<name>`.scale | 1.0 | |
+| `<dwb plugin>`.`<name>`.sum_scores | false | Whether to allow for scores to be sumed up |
+| `<dwb plugin>`.`<name>`.scale | 1.0 | Weight scale |
 
-### obstacle_footprint TrajectoryCritic
+## obstacle_footprint TrajectoryCritic
 
 * `<name>`: obstacle_footprint critic name defined in `<dwb plugin>.critics`
 
 | Parameter | Default | Description |
 | ----------| --------| ------------|
-| `<dwb plugin>`.`<name>`.sum_scores | false | |
-| `<dwb plugin>`.`<name>`.scale | 1.0 | |
+| `<dwb plugin>`.`<name>`.sum_scores | false | Whether to allow for scores to be sumed up |
+| `<dwb plugin>`.`<name>`.scale | 1.0 | Weight scale |
 
-### prefer_forward TrajectoryCritic
+## prefer_forward TrajectoryCritic
 
 * `<name>`: prefer_forward critic name defined in `<dwb plugin>.critics`
 
 | Parameter | Default | Description |
 | ----------| --------| ------------|
-| `<dwb plugin>`.`<name>`.penalty | 1.0 | |
-| `<dwb plugin>`.`<name>`.strafe_x | 0.1 | |
-| `<dwb plugin>`.`<name>`.strafe_theta | 0.2 | |
-| `<dwb plugin>`.`<name>`.theta_scale | 10.0 | |
-| `<dwb plugin>`.`<name>`.scale | 1.0 | |
+| `<dwb plugin>`.`<name>`.penalty | 1.0 | Penalty to apply to backward motion |
+| `<dwb plugin>`.`<name>`.strafe_x | 0.1 | Minimum X velocity before penalty |
+| `<dwb plugin>`.`<name>`.strafe_theta | 0.2 | Minimum angular velocity before penalty |
+| `<dwb plugin>`.`<name>`.theta_scale | 10.0 | Weight for angular velocity component |
+| `<dwb plugin>`.`<name>`.scale | 1.0 | Weight scale |
 
-### twirling TrajectoryCritic
+## twirling TrajectoryCritic
 
 * `<name>`: twirling critic name defined in `<dwb plugin>.critics`
 
 | Parameter | Default | Description |
 | ----------| --------| ------------|
-| `<dwb plugin>`.`<name>`.scale | 0.0 | |
+| `<dwb plugin>`.`<name>`.scale | 0.0 | Weight scale |
 
-### goal_align TrajectoryCritic
+## goal_align TrajectoryCritic
```

Comment:
GoalAlignCritic inherits from GoalDistCritic which inherits from MapGridCritic. 

If we are including parameters from parent classes, then GoalAlignCritic should also include the aggregation_type parameter (from MapGridCritic) 

| `<dwb plugin>`.`<name>`.aggregation_type | "last" | last, sum, or product combination methods |